src/pyird/spec/continuum.py

```diff
@@ -55,7 +55,7 @@ def continuum_oneord(wdata,flat,order):
     continuum = np.interp(wav_tmp[useind],wav_tmp[useind & cutind],continuum)
     return wav_tmp, flux_tmp, ffl_tmp, useind, continuum
 
-def make_blaze(wdata,flat,std_order=None):
+def make_blaze(wdata,flat,RON,std_order=None):
```

Comment:
Can you add the comment for `RON` and consider renaming `RON` to a more user-friendly name?